audiofile/core/info.py

```diff
@@ -161,8 +160,13 @@ def duration(file: str, sloppy=False) -> float:
                     duration = float(duration) / 1000
             except FileNotFoundError:
                 raise binary_missing_error('mediainfo')
-        except OSError:
-            raise broken_file_error(file)
+            # Behavior for broken files is different on Windows
+            # where no error is raised,
+            # but an empty duration is returned.
+            # The error under Windows is then raised
+            # when callinmg 'samples(file)'
+            except subprocess.CalledProcessError:  # pragma: nocover
```

Comment:
Is it not "pragma: no cover"?

Reply:
I guess it works both, at least adding this commit let the coverage go up to 100% on Windows.